tests/NClient/NClient.Standalone.Tests/Clients/ITimeoutClient.cs

```diff
@@ -6,12 +6,15 @@
 namespace NClient.Standalone.Tests.Clients
 {
     [Path("api/timeout")]
+    [Timeout(2)]
     public interface ITimeoutClientWithMetadata : ITimeoutClient
     {
         [GetMethod]
+        [Timeout(1)]
         new int Get(int id);
         
         [GetMethod]
+        [Timeout(1)]
```

Comment:
To separate the tested functionality, it is necessary to have two pairs of methods: with an attribute and without it.

Reply:
Fix